### ISSUE TITLE
Can't download any preconfigured wad anymore

### ISSUE THREAD

**Issue by spaghefoo** [Brawl345/customizemii#72]
When clicking on load after selecting a preconfigured wad(eg: Static Base),
![image](https://user-images.githubusercontent.com/94832681/166113870-de70dd73-5851-446d-a97b-31f528157898.png)
This pops up.

**Comment by Brawl345** [comment]
You are right. Maybe some funky TLS stuff or something. In the meantime, you can get the Base WADs here: https://github.com/Brawl345/customizemii/tree/master/Base_WADs
